applications/CompressiblePotentialFlowApplication/python_scripts/apply_far_field_process.py

```diff
@@ -1,6 +1,6 @@
 import KratosMultiphysics
-import KratosMultiphysics.CompressiblePotentialFlowApplication as CompressiblePotentialFlowApplication
-#from CompressiblePotentialFlowApplication import*
+import KratosMultiphysics.CompressiblePotentialFlowApplication as CPFApp
+import math
```

Comment:
I think there is still some issues about how to separate developments into smaller PRs. In the future, try to avoid multiple PRs on the same class / file. This will save time by avoiding duplicate code review and merge conflicts.

Reply:
I completely agree with this point and will bear it in mind in future PRs.